typst-boxes.typ

```diff
@@ -74,7 +75,8 @@
     fill: backgroundColor,
     stroke: 2pt + strokeColor,
     radius: radius,
-    width: width
+    width: width,
+    breakable: breakable
```

Comment:
Applying breakable on box doesn't work. Changing it to block would be necessary.

Reply:
It is strange, because this code works on my machine. I admit that I sometimes need to adjust the horizontal space if the box start with an equation or a figure, but it works.

Reply:
It doesn't work in the webapp, in [this example](https://typst.app/project/rFqtlWOXCToCTgTPu5vkls).

Reply:
I am currently thinking to rewrite these boxes with `showybox` "under the hood". Doing so would bring more functionality to the boxes.